pytest_splunk_addon_ui_smartx/components/controls/single_select.py

```diff
@@ -52,6 +53,10 @@ def __init__(self, browser, container, searchable=True, allow_new_values=False):
                 "combobox": Selector(
                     select=container.select + ' [data-test="combo-box"]'
                 ),
+                "listbox_role": Selector(select=container.select + ' [role="listbox"]'),
```

Comment:
Listbox is not informative + uses internal implementation. Better to use testing API
```suggestion
                "menu": Selector(select=container.select + ' [data-test="menu"]'),
```